app/src/main/java/pl/droidsonroids/droidsmap/feature/employee/api/EmployeeImageInteractor.kt

```diff
@@ -0,0 +1,35 @@
+package pl.droidsonroids.droidsmap.feature.employee.api
+
+import android.net.Uri
+import com.google.android.gms.tasks.OnCompleteListener
+import com.google.android.gms.tasks.Task
+import io.reactivex.Single
+import io.reactivex.Single.create
+import pl.droidsonroids.droidsmap.base.BaseFirebaseStorageInteractor
+
+
+class EmployeeImageInteractor  : BaseFirebaseStorageInteractor(), EmployeeImageEndpoint {
+
+    override fun setStorageNode() {
+        storageQueryNode = firebaseStorage.reference
+                .child("employees")
+    }
+
+    override fun getEmployeeImageUrl(employeeId: String): Single<String> {
+        return create { emitter ->
+            setStorageNode()
+            val downloadingTask = storageQueryNode.child(employeeId + ".jpg").downloadUrl
```

Comment:
wydaje mi się, ze firebase nie dodaje inferowanych rozszerzeń do nazw, to może nic nie znaleźć